web/templates/settings/admin_emoji_list.hbs

```diff
@@ -10,9 +10,9 @@
             </a>
         </span>
     </td>
-    <td>
+    <td class="panel_user_list">
         {{#if author}}
-        <span class="emoji_author">{{author.full_name}}</span>
+        {{> ../user_display_only_pill  display_value=author.full_name }}
         {{else}}
         <span class="emoji_author">{{t "Unknown author" }}</span>
         {{/if}}
```

Comment:
This seems a simpler approach, also avoids adding `user_id` and `img_src` in `settings_emoji.ts`

```
    <td class="panel_user_list">
        {{#with author}}
        {{> ../user_display_only_pill  display_value=full_name is_active=true img_src=avatar_url}}
        {{else}}
        <span class="emoji_author">{{t "Unknown author" }}</span>
        {{/with}}
    </td>
```

Reply:
This handle the deactivated users as well.

Reply:
there is two problems in your code:
- `else` tag you are using without `if/else` block .
- it was showing activated user as deactivated as well 

![image](https://github.com/user-attachments/assets/652eef11-3867-4056-8b3a-e21313d0d882)

what I have implemented is avoid adding `user_id ` and `img_src `.